src/components/menu-bar/menu-bar.jsx

```diff
@@ -18,6 +18,7 @@ const MenuBar = function MenuBar () {
         >
             <div className={classNames(styles.logoWrapper, styles.menuItem)}>
                 <img
+                    alt="Scratch logo"
```

Comment:
Should this be "Scratch logo" or simply "Scratch"?

Reply:
These tips might be helpful?
https://material.io/guidelines/usability/accessibility.html#accessibility-writing
It's not totally clear, but 'Label what an element does, not what it is' might lean toward 'Scratch' instead of 'Scratch logo'

Reply:
Thanks for pointing this out! I changed to "Scratch" since "logo" isn't necessary